sky/utils/resource_checker.py

```diff
@@ -276,9 +276,10 @@ def get_all_managed_jobs():
         # pylint: disable=import-outside-toplevel
         from sky.jobs.server import core as managed_jobs_core
         try:
-            return managed_jobs_core.queue(refresh=False,
-                                           skip_finished=True,
-                                           all_users=True)
+            filtered_jobs, _, _, _ = managed_jobs_core.queue(refresh=False,
```

Comment:
Hi @seongsukwon-moreh, could you please help add return type `List[Dict[str, Any]]` for the `get_all_clusters` and `get_all_managed_jobs` functions, so that any mismatch changes can be caught later?
Thanks!